### PR TITLE
feat(utils/string_parser): added function to transform csv string to list

### PR DESCRIPTION
**Pull request by daveanthonyc**
Created utils folder with a method to transform csv string to list. 
Also added was a unit test
It will be used within the blog views when parsing incoming csv data.

### CODE REVIEW

File: backend/tests/test_string_parser.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import unittest
+from utils.string_parser import string_parser
+
+class String_Parser_Test(unittest.TestCase):
```
Comment (daveanthonyc) [inline backend/tests/test_string_parser.py]: CamelCase for classes

File: backend/tests/test_string_parser.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    def test_words_are_split(self):
```
Comment (daveanthonyc) [inline backend/tests/test_string_parser.py:5]: Consider more edge cases

**Review by daveanthonyc — COMMENTED** [review]
Please rename classes to have camel cases and add edge cases

**Review by daveanthonyc — COMMENTED** [review]
LGTM
